package.json

```diff
@@ -16,7 +16,7 @@
         "singleQuote": true,
         "semi": false,
         "bracketSpacing": true,
-        "arrowParens": "avoid",
+        "arrowParens": "always",
```

Comment:
Partial solution for https://github.com/aws/aws-toolkit-vscode/issues/3662

Reply:
How does this partially solves the issue? Seems like `arrowParens` only adds parenthesis around the arguments, but the issue is relating to how arrow funcs structure their returns.

Reply:
😅  yeah, it doesn't directly solve https://github.com/aws/aws-toolkit-vscode/issues/3662 . I should have said "this is something I discovered while working on https://github.com/aws/aws-toolkit-vscode/issues/3662 , and since `arrowParens:always` is now the default for what appears to be a solid rationale, I enabled it".